src/App.js

```diff
@@ -1,61 +1,84 @@
 import React from 'react';
+import './App.css'
+import Header from './components/Header'
+import MainBody from './components/MainBody'
+import Footer from './components/Footer'
 
-function App() {
-  return (
-    <section className="todoapp">
-      <header className="header">
-        <h1>todos</h1>
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todoItems: [],
+      userText: '',
+    }
+  };
 
-        <input
-          className="new-todo"
-          placeholder="What needs to be done?"
-          autoFocus=""
-        />
-      </header>
+  onTextFieldChange = (event) => {
+    const eValue = event.target.value;
+    this.setState((prev) => {
+        return {
+            ...prev, userText: eValue
```

Comment:
It is not necessary to spread prevState. You can simplify it.
```suggestion
  this.setState({ userText: eValue })
```

Reply:
Simplified!